lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/utils/RemoteManager.java

```diff
@@ -65,6 +65,7 @@ public void close() throws Exception {
     return this.raw.registerHandler(messageType, theHandler);
   }
 
+  // TODO[REEF-547]: This method uses deprecated raw.registerErrorHandler.
```

Comment:
I need to recall why we deprecated registerErrorHandler. 
@markusweimer Can you help?